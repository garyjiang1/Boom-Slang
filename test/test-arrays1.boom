int[] arr = default int[10]
int i = 0
loop i += 2 while i < 10:
	arr[i] = i
i = 0
loop i += 1 while i < 10:
	println(arr[i])

arr = [0,9,8,7,6,5,4,3,2,1]
println("")
println(arr[7])

def give_me_an_array() returns string[]:
	return ["hi","I","love","PLT","!"]

string[] foo = give_me_an_array()
string[] bar = give_me_an_array()

println("")
i = 0
loop i += 1 while i < 5:
	println(foo[i])
