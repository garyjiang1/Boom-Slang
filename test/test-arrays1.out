0
0
2
0
4
0
6
0
8
0

3

hi
I
love
PLT
!
